include/oboe/Definitions.h

```diff
@@ -469,6 +469,13 @@ namespace oboe {
 
     };
 
+    /**
+     * The time at which the frame at `position` was presented
+     */
+    struct FrameTimestamp {
+        int64_t position;
+        int64_t timestamp;
```

Comment:
position // in frames
timestamp // in nanoseconds

Reply:
fixed